File: PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber {
    public  static  void main(String[] args) {
        Scanner scv = new Scanner(System.in);
        int n = scv.nextInt();

        //int count = 0;
        boolean isPrime = true;

        for(int k = 2; k<n; k++){
            if(n%k==0) {
                // count++; // count = count + 1;
                isPrime = false;
                break;
            }
        }

        // if(count == 0) {
        if(isPrime){
            System.out.println(n + " is prime number!");
        } else {
            System.out.println(n + " is not prime number!");
        }
    }
}

File: PrimeNumber.py
n = int(input("입력해주세요. > "))
isPrime = True

for k in range(2, n):
    if n % k == 0:
        isPrime = False
        break

if isPrime:
    print(n,"is prime number!")
else:
    print(n,"is not prime number!")